cohort-engine-api-web/launchers/cohort-engine-api-web-VT.launch

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?><launchConfiguration type="org.eclipse.jdt.junit.launchconfig">
```

Comment:
I generally prefer not to commit eclipse specific files, but I'm unfamiliar with the purpose of this one so not sure if it could be auto-generated by anyone who wishes to launch in eclipse?

Reply:
This file is used to launch a junit test suite from eclipse with 1 click which is very handy. It can't fully be autogenerated because it would require knowledge of the VM arguments which would be difficult to dig up for someone who isn't very familiar with the code. I would prefer to keep it in, unless it is causing a problem on other IDEs because it is something I use all the time.